README.md

```diff
@@ -148,7 +148,7 @@ tools installed:
 $ sudo apt-get install nodejs npm
 $ mkdir $HOME/opt
 $ cd $HOME/opt
-$ npm install pulp
+$ npm install pulp@4.4.1
```

Comment:
thanks!  this is certainly a good idea.

i would freeze the purescript version as well before merge.  any objections?